demos/list.html

```diff
@@ -125,9 +39,9 @@
     <main>
       <div class="mdc-toolbar-fixed-adjust"></div>
       <section class="hero">
-        <ul class="mdc-list mdc-list--two-line mdc-list--avatar-list two-line-avatar-text-icon-demo">
+        <ul class="mdc-list mdc-list--two-line mdc-list--avatar-list demo-list--two-line-avatar-meta-icon demo-list">
```

Comment:
Nit: demo-list before demo-list--modifierthing?

Reply:
I have brought shame and dishonor to my family. Fixed 😛